dask_cuda/utils.py

```diff
@@ -96,8 +105,13 @@ def get_cpu_count():
 
 @toolz.memoize
 def get_gpu_count():
-    pynvml.nvmlInit()
-    return pynvml.nvmlDeviceGetCount()
+    if _is_tegra():
+        import numba.cuda
+
+        return len(numba.cuda.gpus)
+    else:
+        pynvml.nvmlInit()
+        return pynvml.nvmlDeviceGetCount()
 
 
 def get_cpu_affinity(device_index):
```

Comment:
There is also some `pynvml` calls in this function that we would want to guard against. Currently it calls things like `pynvml.nvmlDeviceGetCpuAffinity` and `pynvml.nvmlDeviceGetHandleByIndex`.

Reply:
Resolved in recent changes